### PR TITLE
Setup matching service backend

### PR DESCRIPTION
**Pull request by larrywang0701**
Setup matching service backend, with some temporary code that will match anyone together for now (in order to test my matching frontend for now)

#48

### CODE REVIEW

File: matching-service/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+### Running the backend server
+ - Run `npm i` in `./matching-service` to install the dependencies.
+ - Use the `.env` file to change `FRONTEND_ADDRESS` and `PORT` if necessary (or the backend will use the default values `FRONTEND_ADDRESS=http://localhost:5173` and `PORT=5000`).
+ - Run `npm run dev` in `./matching-service`. If it works, you will see `Server started. Port = xxxx` in the console.
+
+### How to run the containerized backend server with docker:
+ - 1. Install docker on your device: [Docker](https://www.docker.com)
+ - 2. Setup the `.env` file as the above paragraph mentioned
+ - 3. Run `npm run docker-build-image` to build the image (named `matching-service-image`)
+ - 4. Run `npm run docker-create-container` to create a default container (named `matching-service`). This will create and run the default container. Now the server should be accessible via the port `4000`.
```
Comment (wjayee) [inline matching-service/README.md:15]: i think for matching u decided the port to be 5000 not 4000?

**Review by wjayee — COMMENTED** [review]
js a small typo in the readme but otherwise looks good !

File: matching-service/README.md (hunk 1 above)
Comment (brein62) [reply matching-service/README.md:15]: yea thats fine since 3000 is question service and 4000 is user service iirc

Comment (brein62) [reply matching-service/README.md:15]: so by extension 5000 being matching service works out

**Review by brein62 — APPROVED** [review]
LGTM!

File: matching-service/README.md (hunk 1 above)
Comment (brein62) [reply matching-service/README.md:15]: i think here it's just a typo

Comment (brein62) [inline matching-service/README.md:15]: don't forget to fix the typo here!

**Review by brein62 — CHANGES_REQUESTED** [review]
LGTM! just one more thing

File: matching-service/README.md (hunk 1 above)
Comment (larrywang0701) [reply matching-service/README.md]: fixed

**Review by brein62 — APPROVED** [review]
lgtm! thanks for the fix

**Pull request merged**